connect/client/fluent.py

```diff
@@ -213,6 +213,7 @@ def _get_namespace_class(self):
 
 
 _HTTP_TRANSPORT = httpx.AsyncHTTPTransport()
+_ENV_PROXIES = get_proxy()
```

Comment:
I think it's a better option to let httpx handle such environment variables instead of doing that here.

To make it works, we should just reuse the same ssl context for each `httpx.AsyncClient` instance and let the client instance to build the transport the right way.

Like:

```
_SSL_CONTEXT = httpx.create_ssl_context()
...
self._session = httpx.AsyncClient(verify=_SSL_CONTEXT)
```

Reply:
`httpx.create_ssl_context ` is called inside AsyncHTTPTransport init and when we are initing AsyncClient - ssl context is taken from transport.
But when we are reusing HTTPTransport - flag `allow_env_proxies` is set to False, so environment variables related to proxy are ignored.

Reply:
Please take a look at:

https://github.com/encode/httpx/blob/master/httpx/_client.py#L672
https://github.com/encode/httpx/blob/master/httpx/_client.py#L721
https://github.com/encode/httpx/blob/master/httpx/_client.py#L721
https://github.com/encode/httpx/blob/master/httpx/_config.py#L53
https://github.com/encode/httpx/blob/master/httpx/_config.py#L112

Reply:
Instead of https://github.com/encode/httpx/blob/master/httpx/_client.py#L721 it will go to https://github.com/encode/httpx/blob/master/httpx/_client.py#L714, because transport is not None

Reply:
And why is it not None ? We should not pass a transport to the AsyncClient constructor anymore.